### PR TITLE
feat: add about us and events section

### PR DESCRIPTION
**Pull request by ayussh-2**

### CODE REVIEW

File: src/pages/Playground.jsx [hunk 1]
```diff
@@ -1,8 +1,11 @@
+import About from '../components/about/About'
+
 const Playground = () => {
   return (
     <div>
-      <h1>Playground page</h1>
-      <p>For testing purpose</p>
+      {/* <h1>Playground page</h1>
```
Comment (Srish-ty) [inline src/pages/Playground.jsx]: Remove the h1 and p tags. remove the commented part

**Review by Srish-ty — CHANGES_REQUESTED** [review]
delete `pnpm-lock.yaml` file and commit changes
or `git rm pnpm-lock.yaml`

**Review by Srish-ty — CHANGES_REQUESTED** [review]
@ayussh-2 
make the title fonts smaller after brk-8 for mobile screens

File: src/components/about/aboutCard.styles.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    background-color: ${bgColor};
+  `
+])
+
+export const Section = styled.div`
+  ${tw`text-white-1 flex items-center justify-center flex-col`}
+`
+export const Title = styled.h1(({ isSuperTitle }) => [
+  isSuperTitle
+    ? tw`brk-8:text-[96px] brk-7:text-[96px] text-[60px] mt-20`
```
Comment (Srish-ty) [inline src/components/about/aboutCard.styles.jsx]: make the title font smaller for screens smaller than 538px. 
image

**Comment by Srish-ty** [comment]
@ayussh-2 run `pnpm lint:fix`

**Comment by ayussh-2** [comment]
Oook got it

File: src/components/about/About.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      <CardContainer2>
+        <Title>About Us</Title>
+        <SubSection>
+          <AboutCard
+            title={aiche.title}
+            description={aiche.description}
+            bgColor={aiche.bgColor}
+            isSuperTitle={true}
+          />
+          <Image
```
Comment (anmol5936) [inline src/components/about/About.jsx]: use the link by importing from config file

**Review by anmol5936 — CHANGES_REQUESTED** [review]

**Comment by anmol5936** [comment]
@Shurtu-gal  LGTM 🚀
Please proceed with merging

**Review by anmol5936 — DISMISSED** [review]

**Review by anmol5936 — DISMISSED** [review]

**Review by anmol5936 — DISMISSED** [review]

**Review by anmol5936 — APPROVED** [review]

**Pull request merged**